libs/novui/src/panda-sandbox/css.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import { Meta, StoryFn } from '@storybook/react/*';
+import React from 'react';
+import { css, cx } from '../../styled-system/css';
+
+export default {
+  title: 'Panda Sandbox/css',
+} as Meta;
+
+// css simply outputs a className string
```

Comment:
Could we give an example of what the output string could look like?

Reply:
I'm not really sure that that adds any benefit, and may actually just add unnecessary info

Reply:
Fair point, it's probably irrelevant and subject to changes.